### PR TITLE
pwm_out_sim: don't advertise garbage

### PR DESCRIPTION
**Pull request by julianoes**
Simple fix to delay advertising until we have valid data.

### CODE REVIEW

File: src/drivers/pwm_out_sim/PWMSim.cpp [hunk 1]
```diff
@@ -293,9 +290,15 @@ PWMSim::run()
 				orb_publish_auto(ORB_ID(multirotor_motor_limits), &_mixer_status, &motor_limits, &instance, ORB_PRIO_DEFAULT);
 			}
 
-			/* and publish for anyone that cares to see */
 			_actuator_outputs.timestamp = hrt_absolute_time();
-			orb_publish(ORB_ID(actuator_outputs), _outputs_pub, &_actuator_outputs);
+
+			if (_outputs_pub == nullptr) {
```
Comment (dagar) [inline src/drivers/pwm_out_sim/PWMSim.cpp]: _outputs_pub.publish(_actuator_outputs);

File: src/drivers/pwm_out_sim/PWMSim.hpp [hunk 2]
```diff
@@ -109,7 +109,7 @@ class PWMSim : public cdev::CDev, public ModuleBase<PWMSim>
 
 	int		_armed_sub{-1};
 
-	actuator_outputs_s _actuator_outputs = {};
+	actuator_outputs_s _actuator_outputs {};
 	orb_advert_t	_outputs_pub{nullptr};
```
Comment (dagar) [inline src/drivers/pwm_out_sim/PWMSim.hpp]: uORB::Publication<actuator_outputs_s> _outputs_pub{ORB_ID(actuator_outputs)};

Comment (dagar) [inline src/drivers/pwm_out_sim/PWMSim.hpp]:
```suggestion
	uORB::Publication<actuator_outputs_s> _outputs_pub{ORB_ID(actuator_outputs)};
```

**Comment by julianoes** [comment]
Thanks for the suggestion @dagar, will do.

**Pull request merged**